raiden-ts/src/transfers/reducer.ts

```diff
@@ -223,18 +223,18 @@ function channelCloseSuccessReducer(
   state: RaidenState,
   action: channelClose.success,
 ): RaidenState {
-  return {
-    ...state,
-    sent: mapValues(
-      (v: SentTransfer): SentTransfer =>
-        // if transfer was on this channel, persist CloseChannel txHash, else pass
-        v.transfer[1].channel_identifier.eq(action.payload.id) &&
-        v.transfer[1].recipient === action.meta.partner &&
-        v.transfer[1].token_network_address === action.meta.tokenNetwork
-          ? { ...v, channelClosed: timed(action.payload.txHash) }
-          : v,
-    )(state.sent),
-  };
+  let sent = state.sent;
+  for (const [secrethash, v] of Object.entries(sent)) {
+    if (
+      !v.transfer[1].channel_identifier.eq(action.payload.id) ||
```

Comment:
nitpick: since now we have brackets any chance to extract `v.transfer[1]` to a constant above?

Reply:
Done, thanks ;)